extensions/wikia/CommunityPage/styles/entrypoint/EntryPoint.scss

```diff
@@ -78,7 +71,7 @@ $logo-url: $logo-url-light;
 
 @media #{$breakpoint-below-small-plus} {
 	.WikiaRail {
-		padding-top: 20px;
+		padding-top: $content-spacing-regular;
```

Comment:
I compared how it look after this difference is minor, so I decided to use existing spacing value